platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -181,6 +439,12 @@ int odp_crypto_cipher_capability(odp_cipher_alg_t cipher,
 		src = cipher_capa_null;
 		num = sizeof(cipher_capa_null) / size;
 		break;
+#ifdef __ARM_FEATURE_AES
+	case ODP_CIPHER_ALG_AES_GCM:
+		src = cipher_capa_aes_gcm;
```

Comment:
It is wrong that without __ARM_FEATURE_AES the general crypto capabilities still claim AES-GCM support but when querying the details the number of algorithm variants is actually zero. If AES-GCM is not supported with any key lenght, then the generic crypto capa should say that AES-GCM is not supported. I think validation tests would actually fail because of this. And the same comment for the auth case.

Reply:
> It is wrong that without __ARM_FEATURE_AES the general crypto capabilities still claim AES-GCM support but when querying the details the number of algorithm variants is actually zero. If AES-GCM is not supported with any key lenght, then the generic crypto capa should say that AES-GCM is not supported.
AES-GCM is also protected by __ARM_FEATURE_AES in `odp_crypto_capability` API above. Wouldn't that indicate whether AES-GCM is supported or not? 

>  I think validation tests would actually fail because of this. And the same comment for the auth case.
__ARM_FEATURE_AES has been added to individual cipher and auth crypto capabilities to be consistent with the general crypto capabilities.

Reply:
> AES-GCM is also protected by __ARM_FEATURE_AES in odp_crypto_capability API above. Wouldn't that be itself sufficient to indicate whether AES-GCM is supported or not?

Yes, that is sufficient. I somehow managed to miss it. Sorry about that. You can ignore this comment.